src/elm/Transfer.elm

```diff
@@ -45,15 +46,30 @@ type alias CommunityArgs =
 
 
 type alias Transfer =
-    { to : Eos.Name
-    , from : Eos.Name
+    { id : Int
+    , toId : Eos.Name
+    , fromId : Eos.Name
```

Comment:
Remove both of those (`toId` and `fromId`), we will add details in another nested Map